.circleci/config.yml

```diff
@@ -7,7 +7,8 @@ jobs:
   build:
     working_directory: ~/repo/
     docker:
-      - image: circleci/node:10-browsers
+#      - image: circleci/node:10-browsers
+      - image: ovix/mtc_circleci:latest
```

Comment:
@GuyHarwood - The modified circleci image has produced a green build.

Reply:
great. can we maintain that within MTC/STA org?